From 30b7745d5efaa7857ca4ad45e6b76e8638552fe6 Mon Sep 17 00:00:00 2001
From: Mingliang Liu <liuml07@apache.org>
Date: Thu, 23 Mar 2017 18:06:46 -0700
Subject: [PATCH 2416/2848] HADOOP-14227. S3Guard: ITestS3AConcurrentOps is
 not cleaning up test data. Contributed by
 Mingliang Liu

(cherry picked from commit ec6376485e44f05712c28256fd30094616dccae3)

Change-Id: I27d3063cb519b49ddf0fb6bb085edc08b51759dc
---
 .../hadoop/fs/s3a/scale/ITestS3AConcurrentOps.java |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/scale/ITestS3AConcurrentOps.java b/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/scale/ITestS3AConcurrentOps.java
index b4d3862..e320bb2 100644
--- a/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/scale/ITestS3AConcurrentOps.java
+++ b/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/scale/ITestS3AConcurrentOps.java
@@ -107,7 +107,7 @@ private S3AFileSystem getRestrictedFileSystem() throws Exception {
 
   private S3AFileSystem getNormalFileSystem() throws Exception {
     S3AFileSystem s3a = new S3AFileSystem();
-    Configuration conf = new Configuration();
+    Configuration conf = createScaleConfiguration();
     URI rootURI = new URI(conf.get(TEST_FS_S3A_NAME));
     s3a.initialize(rootURI, conf);
     return s3a;
@@ -115,6 +115,7 @@ private S3AFileSystem getNormalFileSystem() throws Exception {
 
   @After
   public void teardown() throws Exception {
+    super.teardown();
     if (auxFs != null) {
       auxFs.delete(testRoot, true);
     }
-- 
1.7.9.5

